frontend/src/main/scala/bloop/engine/Interpreter.scala

```diff
@@ -571,6 +571,7 @@ object Interpreter {
                   mainClass,
                   cmd.args.toArray,
                   cmd.skipJargs,
+                  envVars = Nil,
```

Comment:
There is no way currently to add env variables inside the CLI and I am not sure about how to add it with a proper user experience. We could have additiona `-E` as in case  of properties.
`-E-TEST=Hello`

But I am not sure. @jvican What do you think?

Reply:
Environment variables are propagated from the terminal where the bloop CLI is used thanks to Nailgun, so there's no need to pass them explicitly through a flag. Users can just export an environment variable and access it from a project run with the CLI.

Reply:
Ach, you are right. This works under bash, but not in fish. Most likely not related then.